src/servers/signal.cc

```diff
@@ -115,9 +120,12 @@ SignalHandler(int signum)
 TRITONSERVER_Error*
 RegisterSignalHandler()
 {
-  // Trap SIGINT and SIGTERM to allow server to exit gracefully
+  // Trap SIGINT, SIGTERM, SIGSEGV and SIGABRT to allow server to exit
+  // gracefully
   signal(SIGINT, SignalHandler);
   signal(SIGTERM, SignalHandler);
+  signal(SIGSEGV, SignalHandler);
```

Comment:
For SEGV and ABRT we don't want to use the existing signalhandler that tries to exit gracefully. Instead we should use a new handler for those that just dumps the stack trace and then does exit(1)

Reply:
Got it, will modify.